static/js/index.js

```diff
@@ -685,8 +714,17 @@ ep_comments.prototype.setComment = function(commentId, comment){
   var comments = this.comments;
   comment.date = prettyDate(comment.timestamp);
   comment.formattedDate = new Date(comment.timestamp).toISOString();
+
   if (comments[commentId] == null) comments[commentId] = {};
   comments[commentId].data = comment;
+
+};
+
+ep_comments.prototype.setCommentReply = function(commentReply){
+  var commentReplies = this.commentReplies;
+  var replyId = commentReply[0];
```

Comment:
it's not very clear what is the structure that comes on `commentReply`. Is it an array? Why do we need to get `commentReply[0]` to get the reply id?